### PR TITLE
Contactpage

### PR DESCRIPTION
**Pull request by iam-marixo**

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>DAY 2 Practice</title>
-  <link href="css/bootstrap.min.css" rel="stylesheet">
-  <link href="css/style.css" rel="stylesheet">
+  <title>DAY 2 Practice</title>  <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+    <link href="css\font-awesome.min.css" rel="stylesheet">
```
Comment (rockygg) [inline contact.html]: `css\font-awesome.min.css` should be `css/font-awesome.min.css`

File: aboutme.html [hunk 2]
```diff
@@ -30,13 +30,13 @@
   <div class="container">
   <div class="carousel-inner" role="listbox">
     <div class="item active">
-      <img src="img/justin-bland-323629.jpg" alt="...">
+      <img src="img\Screenshot (2) redo.png" alt="...">
```
Comment (rockygg) [inline aboutme.html]: the `\` should be `/`

File: aboutme.html [hunk 3]
```diff
@@ -30,13 +30,13 @@
...
     </div>
     <div class="item">
-      <img src="img/sebastian-spindler-221561.jpg" alt="...">
+      <img src="img\Screenshot (5) redo.png" alt="...">
```
Comment (rockygg) [inline aboutme.html]: the `\` should be `/`

File: aboutme.html [hunk 4]
```diff
@@ -30,13 +30,13 @@
...
     </div>
     <div class="item">
-      <img src="img\filip-cernak-1083.jpg" alt="...">
+      <img src="img\Screenshot (4) redo.png" alt="...">
```
Comment (rockygg) [inline aboutme.html]: the `\` should be `/`

**Review by rockygg — APPROVED** [review]
I found a couple places where I think the syntax may effect your site from displaying thing properly. Once those are fixed, I think you're good to merge in

**Comment by iam-marixo** [comment]
so what would i need to do next ?

**Comment by rockygg** [comment]
@iam-marixo you can just merge it in now and send me the link to the live site

**Pull request merged**